_pages/en_US/homebrew-launcher-(browserhax-2020).txt

```diff
@@ -0,0 +1,63 @@
+---
+title: "Homebrew Launcher (BrowserHax 2020)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This method of accessing the Homebrew Launcher takes advantage of some browser oversights in the native 3DS Internet Browser.
+
+{: .notice--success}
+
+### Compatibility Testing
+
+To use this method, your console must be from a compatible region with a compatible browser version.
+
+You can check these details from System Settings.
+
+1. Open System Settings.
+1. In the bottom right of the top screen, you will see a firmware version string in the format of `11.13.0-45x`.
+  + The 45 is your browser version, and the letter is your region
+  + If your browser version is less than 45, you will need to update the console through the System Settings "Software Update" option
+  + E designates EUROPE
+  + U designates USA
+  + J designates JAPAN
+  + If your console has a letter other than any of these three, it is not compatible with this method.
+1. Remember your region designation. You will need to know this later.
+
+### What You Need
+
+* The latest release of [the Homebrew Launcher](https://github.com/fincs/new-hbmenu/releases/latest)
+* The latest release archive of BrowserHax [[for Old 3DS](https://github.com/zoogie/old-browserhax/releases/latest) / [for New 3DS](https://github.com/zoogie/new-browserhax/releases/latest)]
+  + New/Old refers to the model of your console. If you are unsure of which model you have, consult [this image](https://3ds.eiphax.tech/pic/newold.png)
+* An active WiFi connection on your console
+
+### Instructions
+
+#### Section I - Prep Work
+
+1. Copy `boot.3dsx` from the Homebrew Launcher release to the root of your SD card
+  + The root of the SD card refers to the initial directory on your SD card where you can see the Nintendo 3DS folder, but are not inside of it
+1. Copy `browserhax_hblauncher_ropbin_payload.bin` and `arm11code.bin` from the folder corresponding to your console's region to the root of your SD card
+
+#### Section II - BrowserHax
+
+1. Open [this image](https://3ds.eiphax.tech/pic/hax.png) in a new tab or window on your computer
+1. Power on your console and ensure it is connected to an active WiFi network
+1. From the HOME Menu, press the Left Shoulder and Right Shoulder buttons together
+  + This should open a QR Scanner. If it does not, USM (the next part of getting custom firmware) will not work for you. You will need to use a [legacy method](legacy-methods)
+  + If you need to type out the URL, it is `https://zoogie.github.io/web/nbhax/`
+1. Use the QR scanner to scan the QR code you opened in the first step
+1. Tap the "PROCEED TO HAXX" button
+1. Wait for the process to complete
+  + If it displays a yellow screen, power your console off, then try this section again
```

Comment:
'If the console displays a yellow screen' or 'If you get stuck / see a yellow screen'. Very minor nitpick though.